### PR TITLE
AI Training new tutorial: Extract 3D model from video

### PR DESCRIPTION
**Pull request by hyu9a**

### CODE REVIEW

**Comment by OVH-Team-Guides** [comment]
Hello @hyu9a , I tied aligning the content to our usual layout, please check if the hierarchy is still good.
Please also populate the "Requirements" part with more information and links to the proper related pages (see other AI tutorials for reference)

File: pages/public_cloud/ai_machine_learning/training_tuto_11_3d_model_from_video/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+perform High-Fidelity Neural Surface Reconstruction.
+Starting from a video of a scene or an object, it recovers 3D surface structures from the video keyframes through an AI
+model.
+
+### This tutorial
+
+This tutorial aims at showing how to run NVIDIA sample use case with OVHcloud AI-Training jobs.
+
+This tutorial scenario is based on Neuralangelo [git repository](https://github.com/NVlabs/neuralangelo) and
+[collab notebook](https://colab.research.google.com/drive/13u8DX9BNzQwiyPPCB7_4DbSxiQ5-_nGF#scrollTo=FUhJIThkeQoi)
```
Comment (MathieuBsqt) [inline pages/public_cloud/ai_machine_learning/training_tuto_11_3d_model_from_video/guide.en-gb.md]:
```suggestion
[colab notebook](https://colab.research.google.com/drive/13u8DX9BNzQwiyPPCB7_4DbSxiQ5-_nGF#scrollTo=FUhJIThkeQoi).
```

**Comment by MathieuBsqt** [comment]
Great tutorial Mathieu! 

I find it very indicative, perhaps lacking a few sentences at times. I mean, we know what to do but don't always know why. 

I have suggested some typo fixes.
Furthermore, Here are a few details I noticed:

- Given that this is a guide/tuto based on NVIDIA models which are not open-source, I think we should check with the legal department to see if they agree to publish this guide. They are very strict and it might be better to make sure that everything is in agreement.

- Moreover, perhaps our contacts at NVIDIA would also like to review this guide before publication. Jihad will try to reach them.

- You will also have to provide to OVHcloud Team Guides the FR translations for:
- - The guide title
- - The guide excerpt

File: pages/public_cloud/ai_machine_learning/training_tuto_11_3d_model_from_video/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+>		-o json \
+>		-g 1 \
+>		-f ai1-1-gpu \
+>		-v neuralangelo-experiments-lego@NEURALANGELO/neuralangelo:/neuralangelo:rw:cache \
+>		docker.io/chenhsuanlin/colmap:3.8 -- \
+ >     		bash -c "cd /neuralangelo && \
+>			bash projects/neuralangelo/scripts/preprocess.sh lego input/lego.mp4 2 object"
+> ```
+>
+> Note: It takes approximately 8 min to run on 1 ai1-1-gpu GPU.
```
Comment (MathieuBsqt) [inline pages/public_cloud/ai_machine_learning/training_tuto_11_3d_model_from_video/guide.en-gb.md]:
```suggestion
> Note: It takes approximately 8 min to run on 1 `ai1-1-gpu` GPU (V100S).
```

**Review by MathieuBsqt — APPROVED** [review]

**Comment by MathieuBsqt** [comment]
Maybe wait for legal teams to approve before merging

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @hyu9a

**Pull request merged**